src/sentry_utils.c

```diff
@@ -529,3 +529,20 @@ sentry__snprintf_c(char *buf, size_t buf_size, const char *fmt, ...)
     va_end(args);
     return rv;
 }
+
+bool
+sentry__check_min_version(sentry_version_t actual, sentry_version_t expected)
+{
+    if (actual.major < expected.major) {
```

Comment:
I guess we are not considering semver breaks here? As we are allowing the major version to be higher than expected.

Reply:
That's right; I wanted to encode only the minimum version bound in this function. But checking for an upper bound (in another function) would also make sense when checking versions at runtime (also because symmetry is awesome). 

But I wasn't in a hurry to add such a function because we are currently only using it for the `libcurl` check, and it seems that `libcurl` has more of a Linux major-bump idea for 8: https://daniel.haxx.se/blog/2021/09/13/heading-towards-curl-eight/ 

I am open to adding an (open interval) upper-bound check to utils, but I am unsure if we'll use it against `libcurl` under that premise. What do you think?

Reply:
If they won’t break anything with a new major, then there is no reason to check for that either :-)